tests/main_test.c

```diff
@@ -16,6 +16,8 @@ int	main(void)
 {
 	printf("input_to_text_tree in testing\n");
 	input_to_text_tree_test();
-	printf("testing complete\n");
+	printf("tokenization in testing\n");
+	parse_command_test();
```

Comment:
if this PR doesn't create a tree, but just tokenizes the input - can we please also name the testing function as tokenization_test or something? parsing sounds too global

Reply:
ok, i changed all parsing mantions to tokenizing. probably i will do separate folder for parsing as i understood we can handle command argument errors in these two defferent steps.

a also shortned command to cmd